torch/nn/modules/conv.py

```diff
@@ -330,15 +339,8 @@ def __init__(self, in_channels, out_channels, kernel_size, stride=1,
             False, _pair(0), groups, bias, padding_mode)
 
     def forward(self, input):
-        if self.padding_mode == 'circular':
-            expanded_padding = ((self.padding[1] + 1) // 2, self.padding[1] // 2,
-                                (self.padding[0] + 1) // 2, self.padding[0] // 2)
-            return F.conv2d(F.pad(input, expanded_padding, mode='circular'),
-                            self.weight, self.bias, self.stride,
-                            _pair(0), self.dilation, self.groups)
-        return F.conv2d(input, self.weight, self.bias, self.stride,
-                        self.padding, self.dilation, self.groups)
-
+        conv2d_forward(input, self.padding_mode, self.padding, self.weight,
```

Comment:
Missing a `return`.
How come there is no tests to catch this?

Reply:
...will fix. didn't run original conv tests

Reply:
There are tests. The merged commit has a red cross shown above. But maybe the open source CI failure signals didn't pass through or something.

Reply:
yea you ignored the tests, i am not sure why